docs-release-notes.md

```diff
@@ -1,3 +1,31 @@
+# September 8, 2023
+
+## App Builder's Resources
+
+- Configure & Connect to an Edge Server: Fix small spelling typo.
+
+## Device Sync
+
+- Configure and Update your Data Model/Update a Data Model: Fix small spelling typo.
+- Configure Sync/Sync Settings: Update queryable fields documentation for Indexed Queryable Fields and Collection Queryable Fields.
+- Handle Errors/Sync Errors: Update queryable fields documentation for Indexed Queryable Fields and Collection Queryable Fields. 
+
+## Authenticate and MAnage Users
+
+- Manage User Sessions: Update Get a User Access Token Over HTTPS steps for clarity.
+
+## Reference
+
+- App Services Admin API: Add missing properties, ``run_as_user_id`` and ``run_as_user_id_script_source``, to the Create a New Function endpoint.
```

Comment:
This is just a nitpick but I'd probably put the "Create a New Function endpoint" at the beginning of this item so the reader can decide if this is relevant to them _before_ reading to the end. Similar to:

```suggestion
- App Services Admin API: Create a New Function endpoint: Add missing properties: ``run_as_user_id`` and ``run_as_user_id_script_source``.
```